sprocs/issues_insert_for_variant.sql

```diff
@@ -25,7 +25,7 @@ BEGIN
     FROM
         variants AS v
         JOIN questions AS q ON (q.id = v.question_id)
-        JOIN pl_courses AS c ON (c.id = q.course_id)
+        JOIN pl_courses AS c ON (c.id = v.course_id)
```

Comment:
LGTM - we want the issue to be associated with the variant's course, not the question's course.